src/app/shared/alert/state/alert.service.ts

```diff
@@ -111,6 +111,24 @@ export class AlertService {
     this.matSnackBar.open(detailedError);
   }
 
+  /**
+   *
+   * Handles HTTP error responses that shows a detailed message in the matSnackBar and provides and action button
+   * that links to an external site (such as the documentation)
+   *
+   * @param error {HttpErrorResponse} error  The HttpErrorResponse return by the failed Http call
+   * @param actionButtonText {string} The text of the action button
+   * @param link {link} The link that will open in a new tab
+   */
+  public detailedSnackBarErrorWithLink(error: HttpErrorResponse, actionButtonText: string, link: string) {
+    this.clearEverything();
+    const detailedError: string = AlertService.getDetailedErrorMessage(error);
+    this.matSnackBar
+      .open(detailedError, actionButtonText)
```

Comment:
You could increase the duration of the snackbar for this case, I think the default is set to 5 seconds; it might be hard to locate and click on the link before it disappears. But I don't feel that strongly about it.